extension/app/ts/background/background.ts

```diff
@@ -291,7 +291,7 @@ export const handlers = new Map<string, SimulationHandler >([
 	['eth_sendRawTransaction', sendRawTransaction],
 	['eth_gasPrice', gasPrice],
 	['eth_getTransactionCount', getTransactionCount],
-	['interceptor_getSimulatedUnsignedTransactions_v1', getSimulatedUnsignedTransactions],
+	['interceptor_getSimulationResults', getSimulationResults],
```

Comment:
Consider: `eth_getSimulationStack` or `eth_getTransactionStack`?  This query gives both the simulation stack **and** the results of that simulation and callers may not be specifically looking for results which could make this name confusing to them.

Reply:
I don't really want to call it `eth_` because it's not an eth standard. How about `interceptor_getSimulationStack`?

Reply:
Sounds good to me.  It was my mistake to switch to `eth_`, my intent was to comment on the `SimulationResults` bit and my brain just switched to `eth_` automatically.  😖